### PR TITLE
Fix EIAnalyticsHorizontalBarChart granularity issue

### PR DESCRIPTION
**Pull request by RolandHewage**

### CODE REVIEW

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx [hunk 1]
```diff
@@ -88,6 +88,7 @@ class EIAnalyticsHorizontalBarChart extends Widget {
         this.handlePublisherParameters = this.handlePublisherParameters.bind(this);
         this.handleGraphUpdate = this.handleGraphUpdate.bind(this);
         this.handleStats = this.handleStats.bind(this);
+
```
Comment (Niveathika92) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx]: Remove unnessary line

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx [hunk 2]
```diff
@@ -136,7 +137,9 @@ class EIAnalyticsHorizontalBarChart extends Widget {
                     .replace('{{componentType}}', graphType)
                     .replace('{{tenantId}}', TENANT_ID)
                     .replace('{{timeFrom}}', this.state.timeFromParameter)
-                    .replace('{{timeTo}}', this.state.timeToParameter);
+                    .replace('{{timeTo}}', this.state.timeToParameter)
+                    .replace('{{timeUnit}}', this.state.timeUnitParameter);
+
```
Comment (Niveathika92) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx]: Same

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/resources/widgetConf.json [hunk 3]
```diff
@@ -46,4 +46,4 @@
       }
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (Niveathika92) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/resources/widgetConf.json]: Do not commit files not pertaining to the fix

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx (hunk 1 above)
Comment (RolandHewage) [reply components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx]: Fixed in [ f950da9 ](https://github.com/wso2/analytics-solutions/commit/6f41d5f46cc3f07dc89edbcadc0e34da872c23b8)

**Pull request merged**